src/otx/core/exporter/base.py

```diff
@@ -211,3 +220,15 @@ def _embed_openvino_ir_metadata(ov_model: openvino.Model, metadata: dict[tuple[s
             ov_model.set_rt_info(data, list(k))
 
         return ov_model
+
+    def _patch_model(self, model):
+        if self.custom_forward:
+            funcType = types.MethodType
+            model.forward = funcType(self.custom_forward, model)
+
+    def _update_output_names(self):
+        if "output_names" not in self.onnx_export_configuration:
+            self.onnx_export_configuration["output_names"] = ["logits", "saliency_map"]
```

Comment:
Not all the model are exported via onnx, so that statement may have no effect.
As a universal way (for OVC and torch.onnx), one can return a dict with the required keys from the custom forward.

Reply:
Fixed, please take a look